app/src/main/java/com/example/parkingsystem/mvp/contract/ParkingReservationContract.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.parkingsystem.mvp.contract
+
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import java.util.*
```

Comment:
Try to avoid using wildcard imports, check out this post to edit it on Android Studio (i think you already did this part, so it must be that in the configuration there is an option adding wildcards to java utils, try to leave thos options like the pictures)
![Screen Shot 2021-09-07 at 09 27 03](https://user-images.githubusercontent.com/61156888/132344343-db93da5c-5a31-4538-a7eb-3d7826bab3e0.png)


![Screen Shot 2021-09-07 at 09 26 38](https://user-images.githubusercontent.com/61156888/132344258-4919ed5f-d039-453b-8d28-7ad38666ca5f.png)

Reply:
Thanks for the information =)